evap/evaluation/models.py

```diff
@@ -1028,10 +1028,15 @@ class RatingAnswerCounter(Answer):
     bipolar: -3, -2, -1, 0, 1, 2, 3; where a lower absolute means more agreement and the sign shows the pole
     yes / no: 1, 5; for 1 being the good answer"""
 
+    id = models.UUIDField(primary_key=True, default=uuid.uuid4, editable=False)
+
     answer = models.IntegerField(verbose_name=_("answer"))
     count = models.IntegerField(verbose_name=_("count"), default=0)
 
     class Meta:
+        # prevent ordering by date for privacy reasons (getting the objects without explicit ordering
+        # may return the items in insertion order). Also, we run postgres
```

Comment:
yeah, I stopped writing there and asked you what the command was - turns out I forgo that I wanted to write it down here and now I forgot the command again. Luckily, I wrote it down in the issue :D

Will wait how the other discussion goes, maybe this will be removed.